src/genome_builds.php

```diff
@@ -132,68 +132,37 @@
-                  AND COLUMN_NAME IN (?,?)',
-                array(
-                    'position_g_mrna_start_' . $_POST['column_suffix'],
-                    'position_g_mrna_end_' . $_POST['column_suffix']))->fetchAllColumn();
-
+            // Prepare an array to more easily add the required columns into the
+            //  VOG and transcripts tables.
             $aTablesAndTheirColumns = array(
                 TABLE_VARIANTS => array(
-                    '`VariantOnGenome/DNA/' . $_POST['column_suffix'] . '`' => array(
-                        'type' => 'VARCHAR(255)',
-                        'active' => in_array(
-                            'VariantOnGenome/DNA/' . $_POST['column_suffix'],
-                            $aActiveColumnsVariants)
-                    ),
-                    'position_g_start_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_end',
-                        'active' => in_array('position_g_start_' . $_POST['column_suffix'], $aActiveColumnsVariants),
-                    ),
-                    'position_g_end_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_start_' . $_POST['column_suffix'],
-                        'active' => in_array('position_g_end_' . $_POST['column_suffix'], $aActiveColumnsVariants)
-                    ),
+                    'VariantOnGenome/DNA/' . $_POST['column_suffix'] => 'VARCHAR(255)',
+                    'position_g_start_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_end',
+                    'position_g_end_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_start_' . $_POST['column_suffix'],
                 ),
                 TABLE_TRANSCRIPTS => array(
-                    // TODO: Ask Ivo whether or not the mrna positions should be NOT NULL
-                    'position_g_mrna_start_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_mrna_start',
-                        'active' => in_array('position_g_mrna_start_' . $_POST['column_suffix'], $aActiveColumnsTranscripts)
-                    ),
-                    'position_g_mrna_end_' . $_POST['column_suffix'] => array(
-                        'type' => 'INT(10) UNSIGNED AFTER position_g_mrna_end',
-                        'active' => in_array('position_g_mrna_end_' . $_POST['column_suffix'], $aActiveColumnsTranscripts)
-                    ),
+                    'position_g_mrna_start_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_mrna_end',
+                    'position_g_mrna_end_' . $_POST['column_suffix'] => 'INT(10) UNSIGNED AFTER position_g_mrna_start_' . $_POST['column_suffix'],
                 ),
             );
 
-            // Add columns to VOG and Transcripts tables.
-            foreach (array_keys($aTablesAndTheirColumns) as $sTable) {
```

Comment:
Please check the [PHP manual on foreach](https://www.php.net/manual/en/control-structures.foreach.php); if you need both the keys and the values in an array, use:
```php
foreach ($aArray as $sKey => $aVal) {
    ...
}
```
which is much more efficient and keeps the code more readable as well.